app/uk/gov/hmrc/economiccrimelevyreturns/models/CalculatedLiability.scala

```diff
@@ -43,19 +43,31 @@ object Band {
   }
 }
 
-final case class BandRange(from: Long, to: Long)
+final case class EclAmount(amount: BigDecimal, apportioned: Boolean = false)
+
+object EclAmount {
+  implicit val format: OFormat[EclAmount] = Json.format[EclAmount]
+}
+
+final case class BandRange(from: Long, to: Long, amount: BigDecimal)
 
 object BandRange {
   implicit val format: OFormat[BandRange] = Json.format[BandRange]
 }
 
-final case class Bands(small: BandRange, medium: BandRange, large: BandRange, veryLarge: BandRange)
+final case class Bands(
+  small: BandRange,
+  medium: BandRange,
+  large: BandRange,
+  veryLarge: BandRange,
+  apportioned: Boolean = false
+)
```

Comment:
same as reg fe and returns backend - are the defaults to false needed here? this model just reads from the calculator